ClassicAssist/Data/Macros/MacroManager.cs

```diff
@@ -111,7 +111,12 @@ public void Execute( MacroEntry macro )
             {
                 if ( CurrentMacro != null && CurrentMacro.IsRunning )
                 {
-                    if ( CurrentMacro.DoNotAutoInterrupt )
+                    if ( !CurrentMacro.Disableable )
```

Comment:
This stops macros that are marked 'Undisableable' from stopping and restarting when repressing the hotkey...

Why is this wanted?